go.mod

```diff
@@ -49,4 +49,4 @@ require (
 
 replace github.com/gogo/protobuf => github.com/ElrondNetwork/protobuf v1.3.2
 
-replace github.com/ElrondNetwork/arwen-wasm-vm/1_2_7 => github.com/ElrondNetwork/arwen-wasm-vm v1.2.7
+replace github.com/ElrondNetwork/arwen-wasm-vm/v1_3 v1.3.0 => github.com/ElrondNetwork/arwen-wasm-vm v1.2.11-0.20210514111522-e69be2f3a0ae
```

Comment:
do we still need this?

Reply:
It will be replaced with a proper release of Arwen (actual v1.3.0). But `go.mod` will inevitably contain a `replace` statement.